sherpa/astro/tests/test_astro_data2.py

```diff
@@ -816,3 +816,46 @@ def test_pha_invalid_units(invalid, make_test_pha):
         pha.units = invalid
 
     assert str(de.value) == f"unknown quantity: '{invalid}'"
+
+
+def test_pha_reverse_sorted():
+    '''This case for a PHA set with reverse sorted channels is a little
+    non-sensical, but it is a very easy way to hit the problem in #1163.
+
+    Really, we want to test that with a larger test that reads real XMM
+    data, which uses reverse energy ordering (see next test).
+
+    So, if thise tests starts to fail, but the next one passes, this one can
+    probably be removed. It's here now because it's fast, easy, and does not
+    require the large test data.
+    '''
+    d = DataPHA('x', np.array([4, 3, 2, 1, 0]), np.array([5, 4, 3, 2, 1]))
+    d.notice(2, 4)
+    assert len(d.get_dep(filter=True)) == 3
+
+
+@pytest.fixture
+def read_test_image(make_data_path):
```

Comment:
Is this routine used anywhere?

Reply:
No. I wonder how it got here...

Reply:
I know... it's use a lot higher up in the file and I accidentally copied it when I copied one of the other tests to use as template for how to write this one. Fixed.